reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerConfig.java

```diff
@@ -231,6 +240,7 @@ public SslProvider sslProvider() {
 	HttpProtocol[]                                          protocols;
 	int                                                     _protocols;
 	ProxyProtocolSupportType                                proxyProtocolSupportType;
+	boolean                                                 redirectHttp2Https;
```

Comment:
Can we name redirectHttp2Https either redirectHttp or redirectHttpToHttps so that it is not misinterpreted as HTTP/2 protocol.

Reply:
updated to 'redirectHttpToHttps'